build.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/sh
 MAUTRIX_VERSION=$(cat go.mod | grep 'maunium.net/go/mautrix ' | awk '{ print $2 }' | head -n1)
 GO_LDFLAGS="-s -w -X main.Tag=$(git describe --exact-match --tags 2>/dev/null) -X main.Commit=$(git rev-parse HEAD) -X 'main.BuildTime=`date -Iseconds`' -X 'maunium.net/go/mautrix.GoModVersion=$MAUTRIX_VERSION'"
+
+if [ "$(uname)" = 'Darwin' ]; then
+	export CGO_CFLAGS="-I/opt/homebrew/include"
+	export CGO_LDFLAGS="-L/opt/homebrew/lib"
+fi
```

Comment:
:thinking: https://github.com/mautrix/imessage/blob/master/build.sh#L4-L7 is the proper way I think, probably should be a separate commit though